tests/test_form.py

```diff
@@ -0,0 +1,139 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car, Driver
+from taxi.forms import (
+    CarSearchForm,
+    DriverCreationForm,
+    DriverLicenseUpdateForm,
+    DriverSearchForm,
+)
+
+
+class TestCarSearchForm(TestCase):
+    def setUp(self):
+        self.manufacturer = Manufacturer.objects.create(
+            name="test_manufacturer", country="test_country"
+        )
+        self.car1 = Car.objects.create(
+            model="car1_test", manufacturer=self.manufacturer
+        )
+        self.car2 = Car.objects.create(
+            model="car2_test", manufacturer=self.manufacturer
+        )
+        self.car3 = Car.objects.create(
+            model="CAR1_test", manufacturer=self.manufacturer
+        )
+
+    def test_search_car_by_model(self):

```

Comment:
u are testing querysets filtering here, the same is for other tests
it is better to test ur search features through views